### PR TITLE
Dashboard tweaks

### PR DESCRIPTION
**Pull request by renziver**
redesign dashboard (Issue #617 )

screen shot 2017-07-04 at 3 11 19 pm
screen shot 2017-07-04 at 3 11 22 pm

### CODE REVIEW

File: app/assets/stylesheets/components/logo.scss [hunk 1]
```diff
@@ -31,6 +32,7 @@
   @media screen and (max-width: 992px) {
     .logo-img {
       height: 60px;
+      border-radius:50%;
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/logo.scss]: Can we remove this change too please, RE: logo styles

File: app/assets/stylesheets/components/logo.scss [hunk 2]
```diff
@@ -40,6 +42,7 @@
     }
     .logo-img {
       height: 60px;
+      border-radius:50%;
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/logo.scss]: Can we remove this change too please, RE: logo styles

File: app/views/users/_profile_card.html.erb [hunk 3]
```diff
@@ -5,7 +5,7 @@
         <div class="row">
           <div class="col-md-5 align-items-center text-center">
             <div class="user-avatar">
-              <%= image_tag gravatar_url(user, 80), alt: 'gravatar' %>
+              <%= image_tag 'odin-logo.svg', class: 'logo-img' %>
```
Comment (KevinMulhern) [inline app/views/users/_profile_card.html.erb]: can we replace `'odin-logo.svg'` with `@user.image` please? that will use the users avatar pulled in from github.

File: app/views/users/_profile_card.html.erb [hunk 4]
```diff
@@ -17,7 +17,14 @@
             <div class="learning-goal-wrapper">
               <p class="camel bold learning-goal-title">Learning Goal</p>
               <p class="learning-goal">
-                <%= user.learning_goal || 'Not Specified' %>
+                <%= user.learning_goal || ''  %>
```
Comment (KevinMulhern) [inline app/views/users/_profile_card.html.erb]: I think we should do this a bit differently. if the user.learning goal isn't set it will return nil so it will use the value at the other side of the or pipes `||`

so we could do something like this:
```
<%= user.learning_goal || link_to "Settings", edit_user_registration_path, class: 'accent'  %>
```

But I think we should include a bit more than just "settings" if the user doesn't have a learning goal set.
maybe something like "set a learning goal in your settings"

The solution to this imo is to create a helper method in the user_helper.rb file:
```
def set_learning_goal
  <%= link_to "set a learning goal in your settings", edit_user_registration_path, class: 'accent' %>
end
```

and then use it in the view like so:
```
<%= user.learning_goal || set_learning_goal  %>
```

**Comment by KevinMulhern** [comment]
Hey @renziver, this is on the right track 😄 please take a look at the comments I've left. If you need any clarification on any of them please let me know.

File: app/assets/stylesheets/components/user_settings.scss [hunk 5]
```diff
@@ -13,7 +13,6 @@
   .profile-pic {
     margin-right: 32px;
     img {
-      border-radius: 50%;
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/user_settings.scss]: @renziver why was this removed? its needed for the avatar on the settings page. Can we put it back in please?

File: app/assets/stylesheets/components/navbar.scss [hunk 6]
```diff
@@ -53,7 +53,6 @@
       margin-top: 37.8px;
       width: 30px;
       height: 30px;
-      @include border-radius(50%);
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/navbar.scss]: Why was this removed @renziver? its needed to style the users avatar correctly on the navbar. Can we put it back in please?

File: app/helpers/users_helper.rb [hunk 7]
```diff
@@ -7,5 +7,8 @@ def gravatar_url(user, size)
   def lesson_time(user, lesson)
     user.lesson_completion_time(lesson).to_formatted_s(:long)
   end
-end
 
+  def set_learning_goal
```
Comment (KevinMulhern) [inline app/helpers/users_helper.rb]: Can your write a unit test for this please? 😄

File: app/models/user.rb [hunk 8]
```diff
@@ -70,7 +70,7 @@ def send_confirmation_instructions
   private
 
   def default_image(size)
-    "http://www.gravatar.com/avatar/436053b3e050d4156773bc04cfb167fe?s=#{size}"
+    "https://www.gravatar.com/userimage/74417267/598000a0a8cad11e55334075bab42658?s=#{size}"
```
Comment (KevinMulhern) [inline app/models/user.rb]: What image does this link to?

**Comment by KevinMulhern** [comment]
Thanks for making those changes @renziver, this is almost perfect 😄 please take a look at my most recent comments.

File: app/models/user.rb (hunk 8 above)
Comment (renziver) [inline app/models/user.rb]: I used the newer version of the odin project's logo

**Comment by KevinMulhern** [comment]
Your changes have been deployed to our staging site @renziver 
https://staging-odin-project.herokuapp.com/dashboard

Thanks again for contributing.
